Makefile

```diff
@@ -65,6 +69,20 @@ docker-test-unit: $(DOCKER_CACHE)
 	@$(DOCKER_ENV) make test-unit
 	@chmod -R 0755 $<
 
+.PHONY: docker-test-integration
+docker-test-integration:
```

Comment:
This should be part of a separate PR, this PR is large enough as it is.

Reply:
I used docker-test-integration for hew route_test.go which changes route tables hence only implemented when test is running is container. Only it is Ok this be part of PR, but separate commit.